Sources/MusicXML/Complex Types/Sound.swift

```diff
@@ -44,7 +44,7 @@ public struct Sound {
     public let offset: Offset?
     public let midi: [MIDI]
 
-    public init(tempo: Double? = nil, dynamics: Double? = nil, dacapo: Bool? = nil, segno: String? = nil, dalsegno: String? = nil, coda: String? = nil, tocoda: String? = nil, divisions: Divisions? = nil, forwardRepeat: Bool? = nil, fine: String? = nil, timeOnly: TimeOnly? = nil, pizzicato: Bool? = nil, pan: Double? = nil, elevation: Double? = nil, damperPedal: YesNoNumber? = nil, softPedal: YesNoNumber? = nil, sostenutoPedal: YesNoNumber? = nil, offset: Offset? = nil, midi: [MIDI]) {
+    public init(tempo: Double? = nil, dynamics: Double? = nil, dacapo: Bool? = nil, segno: String? = nil, dalsegno: String? = nil, coda: String? = nil, tocoda: String? = nil, divisions: Divisions? = nil, forwardRepeat: Bool? = nil, fine: String? = nil, timeOnly: TimeOnly? = nil, pizzicato: Bool? = nil, pan: Double? = nil, elevation: Double? = nil, damperPedal: YesNoNumber? = nil, softPedal: YesNoNumber? = nil, sostenutoPedal: YesNoNumber? = nil, offset: Offset? = nil, midi: [MIDI] = []) {
```

Comment:
There might be 0 `midi-device`s and/or `midi-instrument`s. http://usermanuals.musicxml.com/MusicXML/MusicXML.htm#CT-MusicXML-sound.htm?Highlight=sound

So I default this to `[]` to avoid needing to set it explicitly when `Sound` does not have any `midi-devices` or `midi-instrument`